tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -229,10 +229,10 @@
     EOF
 
 - name: Create OpenStackDataPlaneNodeSet
+  no_log: "{{ use_no_log }}"
   ansible.builtin.shell: |
     {{ shell_header }}
-    {{ oc_header }}
-    oc apply -f - <<EOF
+    cat <<EOF > edpm-crd.yaml
```

Comment:
This edpm-crd.yaml file is then not used anywhere below to actually create the NodeSet? I think we need an `oc apply` call that would use this file?